packages/sdk-react/src/core/utils/getLegacyAPIErrorMessage.test.ts

```diff
@@ -0,0 +1,57 @@
+import { getLegacyAPIErrorMessage } from '@/core/utils/getLegacyAPIErrorMessage';
+
+describe('getMessageInError', () => {
+  it('should return the message from an error object', () => {
```

Comment:
Can I ask to add more understandable test names to avoid duplicates (line 16)?

Reply:
Thank you, fixed, and improved others.